MAUI/ComboBox/AutoSizing.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: AutoSizing in .NET MAUI ComboBox control | Syncfusion
+description: Learn all about autosizing support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more here.
+platform: maui
+control: SfComboBox
+documentation: ug
+---
+
+# AutoSizing in .NET MAUI ComboBox (SfComboBox)
+
+AutoSizing can be enabled in [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) control so that the control will extend its layout based on the input content.
```

Comment:
AutoSizing can be enabled in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) control so that the control will extend its layout based on the input content.